internal/controller/services/monitoring/resources/data-science-prometheus-network-policy.tmpl.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: data-science-prometheus-secure-access
+  namespace: {{.Namespace}}
+  labels:
+    platform.opendatahub.io/part-of: monitoring
+  ownerReferences:
+    - apiVersion: {{.Component.APIVersion}}
+      kind: {{.Component.Kind}}
+      name: {{.Component.Name}}
+      uid: {{.Component.UID}}
```

Comment:
Wouldn;t we expect these owner references to be added by the template deployment mechanism?